hack/build-install-yaml.sh

```diff
@@ -39,7 +39,7 @@ cat << EOF >> release/standard-install.yaml
 #
 EOF
 
-for file in `ls config/webhook/*.yaml config/crd/experimental/*.yaml`
+for file in `ls config/webhook/*.yaml config/crd/experimental/*.yaml | grep -v kustomization.yaml`
```

Comment:
Tiny nit, I think it would be slightly cleaner to copy what we're doing here: https://github.com/kubernetes-sigs/gateway-api/pull/1203/files#diff-dd99ba6337b056eec54351c4eac4754058316dbc76834e661091b7108deddd55R75.

```suggestion
for file in `ls config/webhook/*.yaml config/crd/experimental/gateway*.yaml`
```

Reply:
All set